docs/articles/nunit/writing-tests/attributes/cancelafter.md

```diff
@@ -0,0 +1,52 @@
+# CancelAfter
+
+Normally, NUnit simply runs tests and waits for them to terminate—the test is allowed to run indefinitely. For certain kinds of tests, however, it may be desired to specify a timeout value.
+
+For .NET Core and later, [`Thread.Abort`](https://learn.microsoft.com/en-us/dotnet/api/system.threading.thread.abort?view=net-7.0) as used by the [`TimeoutAttribue`](./timeout.md) can no longer be used and there is no way to interrupt a endless loop.
+
+For all tests, one could use the `--blame-hang(-timeout)`  options of [`dotnet test`](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-test#options)
+however this will stop any further exection of the remaining tests.
```

Comment:
```suggestion
For all tests, one could use the `--blame-hang(-timeout)`  options of [`dotnet test`](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-test#options). However, this will stop any further exection of the remaining tests.
```